graphstorm-processing/graphstorm_processing/distributed_executor.py

```diff
@@ -327,7 +327,7 @@ def run(self) -> None:
                 try:
                     # Upload existing output files before trying to re-partition
                     if self.filesystem_type == FilesystemType.S3:
-                        self._upload_output_files(loader, force=True)
+                        self._upload_output_files(self.loader, force=True)
```

Comment:
This was the bug that would cause repartition on leader to fail when the output is on S3.

Reply:
Just want to double check, does the error only occur on emr ?

Reply:
No it would occur any time S3 is used with `--do-repartition True`